dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/PolicyTaskFutureImpl.java

```diff
@@ -6,7 +6,7 @@
  * http://www.eclipse.org/legal/epl-v10.html
  *
  * Contributors:
- *     IBM Corporation - initial API and implementation
+ *     IBM Corporation - PRESUBMIT API and implementation
```

Comment:
This comment should be left as "IBM Corporation - initial API and implementation"

Reply:
Thanks for catching that.  It was an unintentionally global replace error.  I'll get it fixed before delivery.